### ISSUE TITLE
Missing swiped binary under cmd

### ISSUE THREAD

**Issue by imranhladha** [SwipeWallet/swipechain-node#1]
Node install fails due to missing 'swiped' under /cmd, as referenced within the Makefile

**Comment by ajlarocca** [comment]
I agree. The swiped directory and main.go are missing. Also swipecli does not install. Otherwise, I can't figure it out.

**Comment by tkdlqm2** [comment]
I agree too, Hasn't this bug been fixed yet?
